include/termcolor/termcolor.hpp

```diff
@@ -39,7 +39,13 @@
 
 
 #include <iostream>
+#ifdef __STRICT_ANSI__  // Required when compiling with
```

Comment:
Hm.. Do we really need this in all cases? Maybe just make that workaround only if both `__STRICT_ANSI__` and `__CYGWIN__` are defined?